### PR TITLE
2.0 ReadMe

### PR DESCRIPTION
**Pull request by JohnONolan**
Maybe squash?

Preview: https://github.com/JohnONolan/Ghost/tree/2.0

### CODE REVIEW

**Comment by kirrg001** [comment]
Maybe squash?

🤷🏻‍♀️😆

File: README.md [hunk 1]
```diff
@@ -1,65 +1,107 @@
...
+```
+$ ghost install
+```
 
-- [General Contributor Guide](https://docs.ghost.org/v1/docs/contributing)
-- [Developer Setup Instructions](https://docs.ghost.org/v1/docs/working-with-ghost)
-- [Admin Client development guide](https://docs.ghost.org/v1/docs/working-with-the-admin-client)
+&nbsp;
 
+Check out our [official documentation](https://docs.ghost.org/) for more information about [recommended hosting stack](https://docs.ghost.org/docs/hosting), properly [upgrading Ghost](https://docs.ghost.org/v1/docs/upgrade), or want to or would like to learn about developing your own Ghost [themes](https://themes.ghost.org/docs) or working with [the API](https://api.ghost.org).
```
Comment (vikaspotluri123) [inline README.md]: > , or want to or would like to learn about developing your own ghost themes or working with the API

This sounds weird to me, although I'm not exactly sure why / what would be a better way 🤔 😬

File: README.md [hunk 2]
```diff
@@ -1,65 +1,107 @@
...
+<p align="center">A fully open source, powerful platform for building and running modern publications,<br>
+    we power serious blogs, magazines and journalism from DuckDuckGo to OpenAI & Sky News.</p>
+<hr />
+<p align="center">
+    <a href="https://ghost.org">Ghost.org</a> |
+    <a href="https://ghost.org/features">Features</a> |
+    <a href="https://ghost.org/customers">Showcase</a> |
+    <a href="https://forum.ghost.org">Forum</a> |
+    <a href="https://docs.ghost.org">Documentation</a> |
+    <a href="https://docs.ghost.org/v1/docs/contributing">Contributing</a> |
```
Comment (kirrg001) [inline README.md]: We will merge this PR into the Ghost 2.0 branch. 
All links in master point already to the v1 docs.

This link should not contain `/v1`/.

**Comment by JohnONolan** [comment]
All issues addressed

**Comment by kirrg001** [comment]
Have you rebased against the 2.0 branch?
You probably have to execute `git rebase origin/2.0`, because you named your branch 2.0 on your upstream.

All links on master are already updated to point to docs/v1. The new readme.md update should get merged into the 2.0 branch and all those links will point to latest docs.

**Comment by JohnONolan** [comment]
Ummm these changes are only to readme, not code. My version supersedes whatever is in 2.0 branch.

**Comment by kirrg001** [comment]
I am fine with merging the changes into master, but then all links have to use docs/v1 links.
And another PR is required to update the links again for Ghost 2.0. Because i already went through the repo and updated all the links 🤠

**Comment by ErisDS** [comment]
@kirrg001 The contents of this PR are the correct contents of the readme file for the 2.0 branch and should completely replace what is currently the contents for the 2.0 branch. With this PR many of the docs links have changed or gone away.

**Comment by kirrg001** [comment]
This PR needs a rebase against the 2.0 branch 👍

**Review by ErisDS — APPROVED** [review]
This is ready to go but on reflection should go into master and then get rebased into 2.0? 

Locally I have a manual merge into master, rebase into 2.0 prepped ready to push but @kirrg001 I didn't want to push without a 👍 from you in case you want to manage it differently - can also merge into 2.0 and do a fresh PR for master if preferred.

Either way - the readme should get updated for both streams - Lemme know how I can help!

**Comment by kirrg001** [comment]
I've cherry picked John's commit to master. Somehow the author was changed during this process, not really sure why 🤔 Let me know if you want to change it, i can revert and do it again with the correct author.

Ghost 2.0 branch was rebased with master and links are up-to-date.
Both streams use the new README update. Closing this PR, because the 2.0 branch just had to rebase against master + latest docs links.

**Pull request closed without merging**